Sources/UINotificationView.swift

```diff
@@ -26,14 +26,28 @@ open class UINotificationView: UIView {
     /// Saved to use for resetting the spacing after an image is shown or hidden.
     private let containerStackViewDefaultSpacing: CGFloat = 14
     
-    private lazy var containerStackView: UIStackView = {
-        let stackView = UIStackView(arrangedSubviews: [self.imageView, self.titlesStackView, self.chevronImageView])
+    var _containerStackView: UIStackView?
```

Comment:
This is kind of ugly, but I need to be able to reinitialize the `containerStackView`. That's why I had to move away from using `lazy`.
Let me know if you can think of a better solution.

Reply:
Can't you just make a method that first removes all arranged subviews and then adds the required arranged subviews again? This way we don't need this ugly code I think!

Reply:
I agree that manipulating the arranged subviews would be a cleaner way to do this.

Reply:
Yeah, I thought about this, but then I lose the benefit of having all of the setup for the stackview happen in one place (through the computed property).
Using a second property felt like a lesser evil since it's just a semantic downside instead of a structural one.
@kairadiagne, what's your opinion on this?

Reply:
@kairadiagne, @AvdLee - Cool, I'll change to use a function then.